web/libs/editor/src/stores/types.d.ts

```diff
@@ -22,14 +32,17 @@ type MSTTag = (
   | {
       type: string;
     }
-) &
-  MSTagProps;
+  )
+  & MSTagProps;
 
 type MixinMSTArea = {
   id: string;
   ouid: number;
-  results: RawResult[];
+  results: MSTResult[];
   parentID: string | null;
+  control: object;
+  object: object;
```

Comment:
probably a thing for later - can we further define these beyond `object`?

Reply:
This whole file is a start for typing more and more things, it's missing a lot, so eventually yes, we'll get there (or get rid of MST)